.travis.yml

```diff
@@ -1,2 +1,8 @@
 script:
```

Comment:
Your deployment script runs on every PR & branch build - that means we'd be deploying the code every time any developer adds a commit or raises a PR even if it's not been reviewed yet. Instead we should limit that to just the master branch, and only on branch builds not PRs. 

The easiest solution for that is to take advantage of Travis' `deploy` stage of the lifecycle - there's a [brief description of that here](https://docs.travis-ci.com/user/job-lifecycle/#deploying-your-code) and [more info on doing that for a script here](https://docs.travis-ci.com/user/deployment/script/)